src/components/Button/Button.tsx

```diff
@@ -26,21 +31,28 @@ export type ButtonProps = {
 >;
 
 const colorClasses: Record<ButtonIntent, string> = {
-  neutral: "bg-black text-white dark:bg-white dark:text-black",
+  neutral: "bg-gray-200 text-black dark:bg-gray-700 dark:text-white",
   primary: "bg-primary text-white",
   positive: "bg-green-700 text-white",
   destructive: "bg-red-700 text-white",
 };
 
+const variantClasses: Record<ButtonVariant, string> = {
+  centered: "justify-center",
+  "left-justified": "justif",
```

Comment:
Not sure if you were going for justify-start as left-justified's value, but justif seems to be a typo.

Reply:
Good catch, thanks!